compiler/ballerina-lang/src/main/java/io/ballerina/projects/JarResolver.java

```diff
@@ -114,6 +115,11 @@ public Collection<JarLibrary> getJarFilePathsRequiredForExecution() {
                 PlatformLibraryScope.DEFAULT,
                 getPackageName(rootPackageContext)));
 
+        // Add resources
+        Optional.ofNullable(jBalBackend.codeGeneratedResourcesLibrary(rootPackageContext.packageId()))
+                .ifPresent(library -> jarFiles.add(
+                        new JarLibrary(library.path(), PlatformLibraryScope.DEFAULT,
+                                getPackageName(rootPackageContext))));
```

Comment:
Shall we extract the `getPackageName(rootPackageContext)` to a variable. Its used in two places.